src/nvim/move.c

```diff
@@ -1630,8 +1630,8 @@ void scroll_cursor_bot(int min_scroll, int set_topbot)
 }
 
 /*
- * Recompute topline to put the cursor halfway the window
- * If "atend" is TRUE, also put it halfway at the end of the file.
+ * Recompute topline to put the cursor halfway across the window
+ * If "atend" is TRUE, also put it halfway to the end of the file.
  */
```

Comment:
I've updated the comment style but I'm having a really hard time understanding what the `atend` parameter does exactly so I didn't try rephrasing its description.